pom.xml

```diff
@@ -29,7 +26,7 @@
   <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
     <connection>scm:git:https://github.com/${gitHubRepo}</connection>
     <developerConnection>scm:git:https://github.com/${gitHubRepo}</developerConnection>
-    <tag>xygeni-sensor-1.2</tag>
+    <tag>${scmTag}</tag>
```

Comment:
This is a variable replaced during the release process only, otherwise you publish snapshots with a release version.